File: c/chuong_01_if_else/bai017.cpp
#include <bits/stdc++.h>
using namespace std;

#define lw printf("LOWER")
#define up printf("UPPER")
#define dg printf("DIGIT")
#define sc printf("SPECIAL")

typedef long long ll;

int main() {
    char c; cin >> c;
    if (c <= '9') (c < '0') ? sc : dg;
    else if (c <= 'Z') (c < 'A') ? sc : up;
    else if (c <= 'z') (c < 'a') ? sc : lw;
    else sc;
    return 0;
}

File: python/chuong_01_if_else/bai017.py
def main():
    c = input()
    if c.islower():
        print("LOWER")
    elif c.isupper():
        print("UPPER")
    elif c.isdigit():
        print("DIGIT")
    else:
        print("SPECIAL")

if __name__ == "__main__":
    main()
